deploy/generic-k8s/ecommerce-app/advertisements.yaml

```diff
@@ -38,6 +38,8 @@ spec:
             value: "user"
           - name: DATADOG_SERVICE_NAME
             value: "advertisements-service"
+          - name: DD_TAGS
```

Comment:
Shall we move to the new unified tagging? (`DD_ENV`, instead of the `env` tag)

Reply:
Oh thats a good point. I will incorporate this into the HELM chart.